src/Hintman.hs

```diff
@@ -2,5 +2,29 @@ module Hintman
        ( someFunc
        ) where
 
+import Options.Applicative
```

Comment:
Can you please explicitly write imports from this library?

Reply:
sure, there's quite a few of them though :P